src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -68,7 +107,22 @@
  *   &lt;property name=&quot;ignoreInlineTags&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example:</p>
+ * <pre>
+ *   public class Test {
+ *     /** &#64;see #sum(int a, int b) *&#47; // ok
+ *     public int sum(int a, int b) {
+ *       return a + b;
+ *     }
+ *   }
+ *   public class Test {
+ *     /** Testing class for {&#64;link Example} *&#47; // violation
+ *     // single-line javadoc comment should be multi-line
```

Comment:
please merge this comment with violation comment to something like
`// violation, javadoc should be multiline`

Reply:
ok